mathOperations_hw/index.js

```diff
@@ -0,0 +1,12 @@
+let firstDigit = prompt(`Please enter the first digit.`);
+let secondDigit = prompt(`Please enter the second digit.`);
+
+firstDigit = Number(firstDigit);
+secondDigit = Number(secondDigit);
+
+let addResult = (firstDigit + secondDigit);
+let minResult = (firstDigit - secondDigit);
+let multiResult = (firstDigit * secondDigit);
+let divResult = (firstDigit / secondDigit);
+
+alert(`${firstDigit} + ${secondDigit} = ${addResult}\n${firstDigit} - ${secondDigit} = ${minResult}\n${firstDigit} * ${secondDigit} = ${multiResult}\n${firstDigit} / ${secondDigit} = ${divResult}` );
```

Comment:
template strings have option to  separate it in a few lines

```suggestion
alert(`${firstDigit} + ${secondDigit} = ${addResult}\n
${firstDigit} - ${secondDigit} = ${minResult}\n
${firstDigit} * ${secondDigit} = ${multiResult}\n
${firstDigit} / ${secondDigit} = ${divResult}` );
```